packages/experimental/Expander/windows/ReactNativeExpander/ExpanderViewManager.cpp

```diff
@@ -0,0 +1,133 @@
+        nativeProps.Insert(L"contentVerticalAlignment", ViewManagerPropertyType::String);
+        nativeProps.Insert(L"headerBackground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"headerForeground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"headerForegroundPointerOver", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"headerForegroundPressed", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"headerBorderBrush", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"headerBorderPointerOverBrush", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"headerBorderPressedBrush", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"headerDisabledForeground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"headerDisabledBorderBrush", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"headerBorderThickness", ViewManagerPropertyType::Number);
+        nativeProps.Insert(L"contentBackground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"contentBorderBrush", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronBackground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronForeground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronPointerOverBackground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronPointerOverForeground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronPressedBackground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronPressedForeground", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronBorderThickness", ViewManagerPropertyType::Number);
+        nativeProps.Insert(L"chevronBorderBrush", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronBorderPointerOverBrush", ViewManagerPropertyType::Color);
+        nativeProps.Insert(L"chevronBorderPressedBrush", ViewManagerPropertyType::Color);
+
+        return nativeProps.GetView();
+    }
+
+    void ExpanderViewManager::UpdateProperties(xaml::FrameworkElement const& view,
+        IJSValueReader const& propertyMapReader) noexcept {
+        if (auto expanderView = view.try_as<ExpanderView>()) {
+           expanderView->UpdateProperties(propertyMapReader);
+        }
+        else {
+           OutputDebugStringW(L"Type deduction for ExpanderView failed.");
+        }
+    }
+
+    // IViewManagerWithExportedEventTypeConstants
+    ConstantProviderDelegate ExpanderViewManager::ExportedCustomBubblingEventTypeConstants() noexcept {
+        return nullptr;
+    }
+
+    ConstantProviderDelegate ExpanderViewManager::ExportedCustomDirectEventTypeConstants() noexcept {
+        return [](winrt::IJSValueWriter const& constantWriter) {
+            WriteCustomDirectEventTypeConstant(constantWriter, "onChange");
```

Comment:
Check if removing this line beaks anything - I think `onChange` is registered by default in RNW code